client-frameworks-support/client-support-angular/projects/client-support-angular/src/lib/route/luigi-activated-route-snapshot-helper.ts

```diff
@@ -1,6 +1,7 @@
 import { ActivatedRouteSnapshot } from '@angular/router';
 
-export class LuigiActivatedRouteSnapshotHelper {
+export class
+LuigiActivatedRouteSnapshotHelper {
```

Comment:
```suggestion
export class LuigiActivatedRouteSnapshotHelper {
```

Should not better to have just in one line?

Reply:
I assumed it was probably prettier 'finding the line too long for a one liner'